src/client/inputhandler.cpp

```diff
@@ -143,7 +143,7 @@ bool MyEventReceiver::OnEvent(const SEvent &event)
 	// Remember whether each key is down or up
 	if (event.EventType == irr::EET_KEY_INPUT_EVENT) {
 		const KeyPress keyCode(event.KeyInput);
-		if (keysListenedFor[keyCode]) {
+		if (keyCode && keysListenedFor[keyCode]) {
```

Comment:
It's not directly obvious what `keyCode` does in this case. It is intuitive for `std::optional` and pointer-like classes (`std::unique_ptr` ...). For example, the function definition wouldn't need a comment if the function was called `isValid`.
It's probably a subjective thing. Feel free to close this comment if you disagree.

Reply:
I could also add an `isValid` method in addition/as an alias to `operator bool`.

Reply:
Actually I think I'll just leave a comment here instead: https://github.com/luanti-org/luanti/pull/14964/commits/c05ad9af6dffbef0abfd474fcede3e6cee70f942